64851 Create utility to Purge orphan records

tyndmar opened the pull request:
File changed: asiUpdateEnv.w
added procedure to remove selected orphan records
(expandable as desired)

Commit:
64851 Create utility to Purge orphan records

File changed: asiUpdateEnv.w
added procedure to remove selected orphan records
(expandable as desired)

jay-advantzware reviewed (commented):
Why repeat this so many times vs a procedure - blah & procedure(remove)
Where procedure remove =        {&cFile}.rec_key LT STRING(YEAR(TODAY),"9999") + STRING(MONTH(TODAY),"99"):
        DELETE {&cFile}.
        ASSIGN 
            iDelCount = iDelCount + 1.
        IF etime GT 3600000 THEN LEAVE.

Commit:
64851 Create utility to Purge orphan records

Files changed: asiUpdateENV.w - replaced long-form orphan purge with call to new utility
New files:
util/purgeOrphans.w - visual wrapper for calling orphan purge procs
util/purgeProcs.p - structured procedure to handle all known purges

Commit:
64851 - minor enhancements
